src/main/java/uk/gov/hmcts/divorce/document/model/DocumentType.java

```diff
@@ -121,8 +121,11 @@
     @JsonProperty("nameChangeEvidence")
     NAME_CHANGE_EVIDENCE("Name change evidence"),
 
-    @JsonProperty("noticeOfProceedings")
-    NOTICE_OF_PROCEEDINGS("Notice of proceedings"),
+    @JsonProperty("noticeOfProceedingsApp1")
```

Comment:
You'll need `@JsonAlias("noticeOfProceedings")` or the old cases will start to fail